imap_processing/tests/lo/test_lo_l1b.py

```diff
@@ -0,0 +1,26 @@
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import load_cdf
+from imap_processing.lo.l1b.lo_l1b import lo_l1b
+
+
+def test_lo_l1b():
+    # Arrange
+    de_file = (
+        imap_module_directory
+        / "tests/lo/test_l1a_cdfs/imap_lo_l1a_de_20100101_v001.cdf"
+    )
+    spin_file = (
+        imap_module_directory
+        / "tests/lo/test_l1a_cdfs/imap_lo_l1a_spin_20100101_v001.cdf"
+    )
+    data = {}
+    for file in [de_file, spin_file]:
+        dataset = load_cdf(file)
+        data[dataset.attrs["Logical_source"]] = dataset
+
+    expected_out = "imap_lo_l1b_de_20100101_v001.cdf"
+    # Act
+    output_file = lo_l1b(data)
+
+    # Assert
+    assert expected_out == output_file.name
```

Comment:
Say anything about the shape/contents of the dataset?

Reply:
I was going to wait to add that until I replace a lot of the temporary code, but you're right, I should do that now.